speed-dreams/source-2.2.3/data/data/menu/controlconfigmenu.xml

```diff
@@ -438,7 +438,7 @@
     </section>
 
     <!--SIMULATED DRIVING ASSISTANCE-->
-    <section name="intervention toggle button">
+    <section name="intervention-toggle button">
         <attstr name="type" val="text button"/>
         <attstr name="show box" val="no"/>
         <attstr name="text" val="MOUSE_MIDDLE_BUTTON"/>
```

Comment:
the intervention toggle can't be a key on the keyboard, could you add that to the hint?